constants.js

```diff
@@ -1,22 +1,22 @@
 const constants = {
-  version: '4.0.0',
+  version: '4.0.1',
```

Comment:
Following [semantic versioning](https://semver.org/), MAJOR.MINOR.PATCH, I think this would be 4.1.0 ~~ we can adapt our own version of semvar too as makes sense